### PR TITLE
DryDock Effort: Enabling the skipped test as associated workItem is fixed

### PR DESCRIPTION
**Pull request by jinujoseph**
workItem
http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/529787
seems to be fixed.

Re-enabling tests passed.
@mavasani 
@dotnet/roslyn-ide

### CODE REVIEW

File: src/EditorFeatures/CSharpTest/Diagnostics/RemoveUnnecessaryCast/RemoveUnnecessaryCastTests.cs [hunk 1]
```diff
@@ -816,8 +816,7 @@ static void Main()
     compareTokens: false);
         }
 
-        [WorkItem(545595, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/545595")]
-        [WpfFact(Skip = "529787"), Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnnecessaryCast)]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnnecessaryCast)]
```
Comment (srivatsn) [inline src/EditorFeatures/CSharpTest/Diagnostics/RemoveUnnecessaryCast/RemoveUnnecessaryCastTests.cs]: Don't remove the workitem attribute - it's good to have a historical link to the bug.

Is wpffact not needed here?

Comment (jinujoseph) [inline src/EditorFeatures/CSharpTest/Diagnostics/RemoveUnnecessaryCast/RemoveUnnecessaryCastTests.cs]: workitem added back.
It works with both wpffact and fact. Kept it as Fact to be consistent with rest of the unit tests in the file.

**Comment by srivatsn** [comment]
:+1:

**Comment by DustinCampbell** [comment]
@jinujoseph: What's the status of this PR? Is it still relevant?

**Pull request closed without merging**